### PR TITLE
Run repeat

### PR DESCRIPTION
**Pull request by kjeverson**
closes issue #165 

Introduces a `--repeat` flag, but also supports a `*` notation to repeat tests.

i.e. `pav run test*3` will run `test` 3 times,  as will `pav run --repeat 3 test`

### CODE REVIEW

File: lib/pavilion/plugins/commands/run.py [hunk 1]
```diff
@@ -196,11 +204,21 @@ def _get_tests(self, pav_cfg, args, mb_tracker, build_only=False,
...
+        repeat_re = re.compile(r'(.*)\*(\d+)')
+
+        tests = []
+        for test_name in args.tests:
+            match = re.search(repeat_re, test_name)
+            if match:
+                name, count = match.groups()
+                tests.extend([name]*int(count))
+            else:
+                tests.append(test_name)
```
Comment (Paul-Ferrell) [inline lib/pavilion/plugins/commands/run.py:216]: Make this work in both direction.  
 - 5*test
 - test*5

Comment (Paul-Ferrell) [reply lib/pavilion/plugins/commands/run.py:216]: You can drop the re entirely. 
Just check that '*' in test_name, and then do a single split(test_name, '*', 1).

Comment (Paul-Ferrell) [reply lib/pavilion/plugins/commands/run.py:216]: You also need to add this to the command argument documentation.

**Review by Paul-Ferrell — CHANGES_REQUESTED** [review]

**Comment by kjeverson** [comment]
Still need to drop `<suite_name>.*` support

**Review by Paul-Ferrell — APPROVED** [review]

**Review by Paul-Ferrell — APPROVED** [review]

**Pull request merged**